packages/sane/lib/src/sane_dev.dart

```diff
@@ -1,19 +1,22 @@
 import 'dart:typed_data';
 
+import 'package:logging/logging.dart';
 import 'package:sane/sane.dart';
 
+final _logger = Logger('sane.sane_dev');
```

Comment:
Maybe just use the global logger, and create a class specific method that would add `[SaneDev]` to the message, so it looks like this:

```
[Sane] sane_start()
[SaneDev] sane_start()
```

Wdyt?

Reply:
Tbh, I never used this package before so I'm really not used to it.
But if we want to filter which logs we want to listen for example, it would be simpler if the whole library use the same logger instance 🤔

Reply:
I'm stupid, we could just filter if the logger name starts with `sane` 🤦‍♂️
Wdyt of naming the logger like this:
sane
sane.dev
Etc.